src/dev/impl/DevToys/Strings/cs-CZ/NumberBaseConverter.resw

```diff
@@ -162,4 +162,10 @@
   <data name="OctalLabel" xml:space="preserve">
     <value>Octal</value>
   </data>
+  <data name="ValueInvalid" xml:space="preserve">
+    <value>The current value isn't a valid {0}</value>
```

Comment:
In the RESX editor, I think it would be helpful for the translators to add a comment that explains what {0} is expected to be.